apps/prairielearn/src/tests/utils/auth.ts

```diff
@@ -40,7 +40,7 @@ export async function getConfiguredUser(): Promise<User> {
 }
 
 export async function getOrCreateUser(authUser: AuthUser): Promise<User> {
-  const user = await callValidatedOneRow(
+  const user = await callRow(
     'users_select_or_insert',
     [authUser.uid, authUser.name, authUser.uin, 'dev'],
     z.object({ user_id: IdSchema }),
```

Comment:
This seems to be a case where the sproc returns multiple columns, but only one is used in the schema. I'm wondering if a comment is warranted here to clarify this is intended.

Reply:
I added a comment in 1e05e18.